task_templates/3_pipelines/15_python3_keras_vizai_joblib/model-metadata.yaml

```diff
@@ -19,6 +19,7 @@ typeSchema:
       condition: IN # can be one of IN or NOT_IN
       value: # can be one or multiple of NUM, TXT, IMG, DATE, CAT, DATE_DURATION, COUNT_DICT, GEO
         - IMG
+        - NUM
```

Comment:
We shouldn't make this change.  @drenfr01 lets talk tomorrow

Reply:
kk!

Reply:
FWIW I told @drenfr01 about this change - I could not get it to succeed until I made this change.

Reply:
From what I had seen the model was after another task that converted from img -> num.  I did a bunch of testing on the same dataset and the type schema was working correctly for me.

Reply:
If it's working it's working. Looks like all the models on that project got deleted, so I can't find the one that had that error that prompted us to make the change, but AFAIK there were no other tasks.

Reply:
Yeah I can't reproduce the problem anymore. I will take another crack at it another time, but if it works by uploading to DataRobot and running it (not just running with drum locally - does this ensure no network?) and it succeeds, I think I'm good with it

Reply:
An improvement we need to make is how we handle projects like that one in drum locally, but it is way beyond the scope of this ticket.